runtime/src/bank.rs

```diff
@@ -54,6 +54,8 @@ use std::{
 
 pub const SECONDS_PER_YEAR: f64 = (365.25 * 24.0 * 60.0 * 60.0);
 
+pub const MAX_LEADER_SCHEDULE_STAKES: Epoch = 3;
```

Comment:
this is small for testing?  I thought we were gonna use 16 or 32

Reply:
Yes. Oh, sorry for the lack of comment. I'm intentionally reducing this to cause CI fragile. :D

@rob-solana At my current understanding with a day of code reading/testing, the assumptions that leader_schedule_stakes for past epochs are always present doesn't cause hard errors for validators (no `unwrap` on the data derived from `bank.epoch_stakes`), but might silently stop operating or skip (old) legimate tasks (`unwrap_or`/`unwrap_default`s). So, I'm chasing the possible problem due to new retaining behavior.